ovs/lib/albacontroller.py

```diff
@@ -63,7 +63,11 @@ class AlbaController(object):
     ETCD_NR_OF_AGENTS_KEY = ETCD_MAINTENANCE_KEY + '/nr_of_agents'
 
     @staticmethod
-    def _get_abm_service_name(alba_backend):
+    def get_abm_service_name(alba_backend):
+        """
```

Comment:
It's also used to generate the name when setting up the abm cluster, so at that point, we can't use ```alba_backend.abm_service.service.name```.